src/services/endpoints.ts

```diff
@@ -54,7 +54,7 @@ export const API_ENDPOINT_ATTENDANCE_CONFIGURATION = '/api/v1/time/config';
 // public endpoints
 export const API_ENDPOINT_API_DEFINITION = '/api/v1/api-definition';
 export const API_ENDPOINT_ENABLED_MODULES = '/api/v1/enabled-modules';
-
+export const API_ENDPOINT_HELP = '/api/v1/help/config';
```

Comment:
This is not a public endpoint. Update `API_ENDPOINT_HELP` to `API_ENDPOINT_HELP_CONFIG`